doc/hands_on/hands_on.md

```diff
@@ -0,0 +1,194 @@
+    %jar /buckets/bfsdefault/default/document-files-virtual-schema-dist-0.3.0-SNAPSHOT-s3-0.1.0.jar;
+/
+```
+
+(If you should experience problems check our [troubleshooting section on the virtual schemas page](https://docs.exasol.com/database_concepts/virtual_schema/user_guide.htm))
+
+## Create the Connection
+
+The virtual schema needs to connect to your AWS bucket.
+For that it needs an AWS-access key. 
+Caution: Don't mix this up with your AWS credentials. 
+While the credentials are the username and password you use for login at the AWS Console, the access key is dedicated for machine login.
+You can use the AWS Console to [create an access-key pair](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey).
+
+Now you need to store this access key-pair in your Exasol database so that the Virtual Schema can use it. 
+For that we use a `CONNECTION` definition , that stores your keys securely.
+
+ ```sql
+CREATE CONNECTION S3_CONNECTION
+    TO 'https://YOUR_BUCKET.s3.eu-central-1.amazonaws.com/'
+    USER 'AWS ACCESS KEY'
+    IDENTIFIED BY 'AWS SECRET KEY';
+``` 
+
+The address (`TO`) must have one of these formats:
+
+* `https://BUCKET.s3.REGION.amazonaws.com/KEY`
+* `http(s)://BUCKET.s3.REGION.CUSTOM_ENDPOINT/KEY` (for a local test version of S3)
+
+## Create a mapping Definition
+
+Now we are going to create a [EDML definition][edml-guide] that maps the JSON data to an Exasol table structure.
+We will start simple and only map the title of each book:
+
+```json
+{
+  "$schema": "https://schemas.exasol.com/edml-1.2.0.json",
+  "source": "book-*.json",
+  "destinationTable": "BOOKS",
+  "addSourceReferenceColumn": true,
+  "mapping": {
+    "fields": {
+      "Title": {
+        "toVarcharMapping": {
+        }
+      }
+    }
+  }
+}
+``` 
+
+Note that we used the wildcard `book-*.json` as `source`. 
+The Virtual Schema adapter will concatenate this with the address (`TO`) from the connection to:
+`https://YOUR_BUCKET.s3.eu-central-1.amazonaws.com/book-*.json`.
+
+Save this mapping as file (for example `myMapping.json`) and upload it to BucketFS:
+
+``` shell script
+curl -I -X PUT -T myMapping.json http://w:writepw@<YOUR_DB_IP>:2580/default/
+```
+ 
+## Create the Virtual Schema
+
+Now we can create the Virtual Schema:
+
+```sql
+CREATE VIRTUAL SCHEMA FILES_VS_TEST USING ADAPTER.S3_FILES_ADAPTER WITH
+    CONNECTION_NAME = 'S3_CONNECTION'
+    SQL_DIALECT     = 'S3_DOCUMENT_FILES'
+    MAPPING         = '/bfsdefault/default/myMapping.json';
+```
+
+## First Query
+
+It's time to run our first query!
+So let's get the title of all our books by running:
+
+```sql
+SELECT * FROM FILES_VS_TEST.BOOKS;
+```
+
+![Result of the first query](./result1.png)
+
+## Filtering
+
+Now let's filter on the title:
+
+```sql
+SELECT * FROM FILES_VS_TEST.BOOKS WHERE SOURCE_REFERENCE = 'book-1.json';
+```
+
+![Result of the filtered query](./result2.png)
```

Comment:
You could also put markdown tables for results. But showing how it looks in a SQL client also okay.